src/ir/auth_logic/datalog_ir.h

```diff
@@ -48,6 +50,7 @@ class DLIRAssertion {
   using DLIRAssertionVariantType = std::variant<Predicate, DLIRCondAssertion>;
   explicit DLIRAssertion(DLIRAssertionVariantType value)
       : value_(std::move(value)) {}
+  const DLIRAssertionVariantType& GetValue() const { return value_; }
```

Comment:
nit: I still don't like this. We should definitely create a visitor. :)

Reply:
we have https://github.com/google-research/raksha/issues/345